templates/campaign.tex

```diff
@@ -0,0 +1,100 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Campaign name}{1. Scenario name}{\images/frenzy.pn}
```

Comment:
```suggestion
\addscenariosection{1}{Campaign name}{1. Scenario name}{\images/frenzy.png}
```

Reply:
I think the image is missing?

Reply:
Oh, I just noticed this part is not rendered.

Reply:
Ha, good catch. I only updated the scenario but not the templates :)

Reply:
Maybe I will remove the templates for now and create a separate PR for it